python/clients.py

```diff
@@ -0,0 +1,78 @@
+from helpers import read_master_key, ClientBuilder, create_json_schema
+
+"""
+This file is meant to be run to demonstrate CSFLE in action. Prior to running
+this file, ensure you've run make-local-data-key.py
```

Comment:
I think we should use PEP8 recommendations for module naming for python scripts: https://www.python.org/dev/peps/pep-0008/#package-and-module-names
tl;dr prefer single-word file names and otherwise use underscores

Reply:
Good catch, done!